### DISCUSSION TITLE
pcl整合包下载路径中具有非法字符咋办

### DISCUSSION THREAD

**Discussion by gupiyo114514** [Meloong-Git/PCL#899] [提问] [answered]
整合包安装失败
→ 解压整合包文件失败
→ 路径中具有非法字符。
   在 PCL.ModModpack.UnpackFiles(String InstallTemp, String FileAddress)
   在 PCL.ModModpack._Closure$__6-0._Lambda$__0(LoaderTask`2 Task)
   在 PCL.ModLoader.LoaderTask`2._Closure$__12-0._Lambda$__0()
错误类型：System.ArgumentException

**Comment by LTCatt** [comment, marked as answer]
这实际上是你的整合包压缩包存在问题，压缩包内的名称可能存在中文并与你的系统编码不兼容等等……这个我没办法修 Orz
